pkgs/applications/audio/soundconverter/default.nix

```diff
@@ -0,0 +1,55 @@
+{ pkgs, python3Packages, lib, fetchurl }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "soundconverter";
+  version = "4.0.3";
+
+  src = fetchurl {
+    url = "https://launchpad.net/soundconverter/trunk/${version}/+download/${pname}-${version}.tar.gz";
+    sha256 = "sha256-hzIG/4LD3705erPYvXb7uoRwF9LtKKIKB3jrhpYMsZ0=";
+  };
+
+  # FIXME: Tests still fail with:
+  # 'No GSettings schemas are installed on the system'
+  doCheck = false;
+
+  buildInputs = with pkgs; [
+    gst_all_1.gst-plugins-bad
+    gst_all_1.gst-plugins-base
+    gst_all_1.gst-plugins-good
+    gst_all_1.gst-plugins-ugly
+    gtk3
+    python3Packages.distutils_extra
```

Comment:
Python dependencies belong to `propagatedBuildInputs` or `pythonPath` (the latter is deprecated but fits better with future: https://github.com/NixOS/nixpkgs/pull/102613).

Reply:
Sure.